test/unit/test_exports/test_derived_quantities/test_derived_quantities.py

```diff
@@ -26,14 +26,22 @@ class TestMakeHeader:
     min_vol_1 = MinimumVolume("retention", 2)
     max_vol_1 = MaximumVolume("T", 2)
 
+    @property
+    def my_derv_quant(self):
+        return self._my_derv_quant
+
+    @my_derv_quant.setter
+    def my_derv_quant(self, value):
+        self._my_derv_quant = DerivedQuantities(value)
```

Comment:
Can you not simply create an empty DerivedQuantities object?

Reply:
So, old tests in each method of this class re-assigned `my_derv_quant.derived_quantities` property that is a list, e.g.: 
https://github.com/festim-dev/FESTIM/blob/a484d8d1a30de1a2273431f222e59896d2e4bed0/test/unit/test_exports/test_derived_quantities/test_derived_quantities.py#L17-L33

As I understand, if `F.DerivedQuantities` is now a list itself, than to modify its elements we have to use the list methods. Direct assignment of `[F.DerivedQuantity1,F.DerivedQuantity2]` to `self.my_derv_quant` will make it a list that does not have `F.DerivedQuantities` attributes leading to the test failure. To minimally change tests, I've introduced the @property and its setter, so each re-assignment of  `self.my_derv_quant` introduces a new `F.DerivedQuantities`. Does it sound logically?

Reply:
ok